src/info/mod.rs

```diff
@@ -133,22 +134,12 @@ pub fn build_info(cli_options: &CliOptions) -> Result<Info> {
     repo.object_cache_size_if_unset(4 * 1024 * 1024);
     let repo_path = get_work_dir(&repo)?;
 
-    let loc_by_language_sorted_handle = std::thread::spawn({
-        let globs_to_exclude = cli_options.info.exclude.clone();
-        let language_types = cli_options.info.r#type.clone();
-        let include_hidden = cli_options.info.include_hidden;
+    let size_by_language_sorted_handle = std::thread::spawn({
         let workdir = repo_path.clone();
-        move || {
-            langs::get_loc_by_language_sorted(
-                &workdir,
-                &globs_to_exclude,
```

Comment:
will gengo be able to use `globs_to_exclude` eventually

Reply:
It can be done with `.gitattributes` (see comment in [this thread](https://github.com/o2sh/onefetch/pull/1152/files#r1313841319)). Otherwise it should be pretty trivial to iterate over gengo's analysis results, which maps filepaths to analysis results, and exclude filepaths that match one or more globs.

Again,

## Pros

- Simplify onefetch's code

## Cons

- Obscure usage for those not familiar with gengo